### PR TITLE
feat(shared-data): update module offsets for DVT cradles

### PR DESCRIPTION
**Pull request by sanni-t**
Closes RSS-204

# Overview

DVT cradles have slightly different measurements than EVT ones. We'll be replacing all existing ones in the office with DVT cradles so need to update the present offsets.

# Test Plan

Make sure to test on an OT3 with DVT cradles and aligned gantry system. Make sure to also complete pipette calibration before starting this test. 
- Upload a protocol with labware loaded on temp module, heater-shaker and thermocycler. Include pipetting steps to/from these modules in the protocol. 
- Run LPC without applying any existing offsets, the pipette should go to the right location on module labware. 
- Ideally there should be no additional LPC offsets required but practically there _might_ be some. Note down these LPC offsets

# Changelog

- updated affine transforms in shared data

# Review requests

- Ideally a code review should be sufficient but from past testing we have found important distinctions between theoretical offset vs actual observed offset so we should probably test this (w/ testing plan above) and note down those differences, if any.
- Make sure I've used the correct offsets mentioned in the excel sheet for DVT cradles. There's tests that verify the resulting offsets, so as long as the final offsets are correct, we should be good.

# Risk assessment

Low.

### CODE REVIEW

File: shared-data/module/definitions/3/heaterShakerModuleV1.json [hunk 1]
```diff
@@ -91,57 +91,57 @@
...
       },
       "3": {
         "labwareOffset": [
-          [-1, 0, 0, -3.125],
-          [0, -1, 0, 0.125],
-          [0, 0, 1, -49.275],
+          [1, 0, 0, 0.125],
+          [0, 1, 0, -1.125],
+          [0, 0, 1, -49.325],
           [0, 0, 0, 1]
```
Comment (SyntaxColoring) [inline shared-data/module/definitions/3/heaterShakerModuleV1.json:129]: What's going on with the flippy rotatey parts of these affine transforms (the first 3 columns)? What were they doing before, and what are they doing now?

Comment (sanni-t) [reply shared-data/module/definitions/3/heaterShakerModuleV1.json:129]: The EVT cradles pushed the h/s in diagonally opposite directions on left vs right slots of the deck. While DVT h/s cradles just don't have any x, y offsets on any slots, so no more rotatey transforms.
(offsets summary [here](https://docs.google.com/document/d/15Ddv8n4lg0DffwIo4wKbUJyfzEXqgj403vtt65D_bW8))

**Review by vegano1 — APPROVED** [review]

File: shared-data/module/definitions/3/temperatureModuleV2.json
Comment (SyntaxColoring) [inline shared-data/module/definitions/3/temperatureModuleV2.json:1]: Do you know why the `V1` version of `temperatureModule` doesn't have similar transforms defined? Is it valid to load a Temperature Module GEN1 on a Flex? If not, do we need a ticket to forbid it?

Same question for the Thermocycler GEN1.

**Review by SyntaxColoring — COMMENTED** [review]
Tested a Heater-Shaker and it seemed...*basically* right, as I showed you in Slack? I don't know. I'm not sure if I did pipette calibration properly.

I haven't had a chance to test the other modules yet. I do want to check the Thermocycler, in particular, because the movement extent spreadsheet specifies the offsets in a slightly tricky way. It's against slot 10 instead of slot 7, and there might be something tricky with the labware mating plane.

File: shared-data/module/definitions/3/temperatureModuleV2.json
Comment (sanni-t) [reply shared-data/module/definitions/3/temperatureModuleV2.json:1]: Ya it's invalid to load a Temp Gen1 and Thermo Gen1 on the flex. I think we have a ticket for it either in RLAB or RSS. But if that's not true then we should create one.

**Review by SyntaxColoring — APPROVED** [review]
The new math all looks correct to me, based on information from the DVT Robot Extents spreadsheet.

Other unrelated things keep blocking testing on a real robot. Currently, I suspect the gantry is misaligned on the FlexDVT bot? Its positioning is all over the place. We're debugging this in Slack. The offsets in this PR plausibly matched the physical offsets of modules with their DVT cradles, but I can't verify within ±3 mm or so until that's resolved.

Aside: this way of expressing module offsets that differ between the OT-2 and OT-3 sure is confusing!

**Comment by SyntaxColoring** [comment]
We've agreed to merge this without completing physical testing. Details in RLAB-335.

**Pull request merged**